app/models/manageiq/providers/openshift/container_manager/container_image.rb

```diff
@@ -1,10 +1,48 @@
 class ManageIQ::Providers::Openshift::ContainerManager::ContainerImage < ContainerImage
-  def annotate_deny_execution(causing_policy)
+  def annotate_image(annotations)
     ext_management_system.annotate(
       "image",
       digest,
+      annotations
+    )
+  end
+
+  def openscap_summary
+    failed_rules = openscap_rule_results.where(:result => "fail").group(:severity).count
+    [[['High'], 'Critical', 3],
+     [['Medium'], 'Important', 2],
+     [['Low'], 'Medium', 1],
+     [['Info', 'Unknown'], 'Low', 0]].collect do |severities, label, index|
+      {
+        :label         => label,
+        :severityIndex => index,
+        :data          => failed_rules.select { |sev| severities.include?(sev) }.values.sum
+      }
+    end
+  end
+
+  def security_quality_annotation(compliant)
+    {"quality.images.openshift.io/vulnerability.manageiq" => {
```

Comment:
https://github.com/adellape/openshift-docs/blob/master/security/container_content.adoc#user-content-security-image-metadata
lists 5 "acceptable values" for "providerId", which doesn't include "manageiq".  I'm not sure they really intend this to be a closed list, but if yes perhaps this should be `quality.images.openshift.io/vulnerability.openscap`?